packages/terra-spacer/tests/jest/Spacer.test.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Spacer from '../../src/Spacer';
+
```

Comment:
Add tests with react-bps props passed in.

Reply:
I think it is there in one of the commits below? https://github.com/cerner/terra-core/pull/1011/commits/2aa563b094dccbc9f16b713591122a4c1a42a0b9#diff-e4f682d5d51dfdccee7d210d9dda45daR34